tide_landing_page.install

```diff
@@ -384,3 +384,18 @@ function tide_landing_page_update_8008() {
     $field_config->save();
   }
 }
+
+/**
+ * Summary field in carousal Card event is restricted to 200 characters.
+ */
+function tide_landing_page_update_8009() {
+  /** @var \Drupal\Core\Entity\Display\EntityFormDisplayInterface $entity_form_display */
+  $entity_form_display = Drupal::entityTypeManager()
+    ->getStorage('entity_form_display')
+    ->load('paragraph.card_event.default');
+  $field_paragraph_summary = $entity_form_display->getComponent('field_paragraph_summary');
+  if (!$field_paragraph_summary['third_party_settings'] || !$field_paragraph_summary['third_party_settings']['maxlength']['maxlength_js']) {
```

Comment:
@vincent-gao you need to check with either `isset()` or `!empty()`. Checking `!$var` will result in a PHP warning if `$var` is not set.

The hook_update sets the maxlength settings for existing fields if they are *not set*. If the field already has `$field_paragraph_summary['third_party_settings']['maxlength']`, the hook_update should not overwrite the settings. Your hook_update also needs to set all the settings of maxlength as per the config, not just `maxlength_js`.

Reply:
Thanks, Sonny. if `maxlength_js` already has a value, we don't want to change it. 
fixed other requirements.